README.md

```diff
@@ -101,6 +107,10 @@ This would correspond roughly to a freestyle project whose `config.xml` includes
 
 with no `<hudson.triggers.SCMTrigger>` (polling).
 
-# Supporting Pipeline from an SCM plugin
+## Supporting Pipeline from an SCM plugin
 
 See the [compatibility guide](https://github.com/jenkinsci/pipeline-plugin/blob/master/DEVGUIDE.md#scms).
```

Comment:
I dunno if this is something you want to change "while you're in there," but this link now has only a pointer to [the developer guide for writing pipeline-compatible plugins](https://jenkins.io/doc/developer/plugin-development/pipeline-integration/).

Reply:
Good catch, I'll update it.

Reply:
Done in ccd41d1.